spec/features/authentication/lock_user_account_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'features/support/not_signed_in'
+require 'features/support/sign_up_helpers'
+require 'support/notifications_service'
+require 'support/confirmation_use_case'
+
+describe "Locking a users account after too many failed login attempts" do
+  include_examples 'confirmation use case spy'
+  include_examples 'notifications service'
+
+  after { Timecop.return }
+
+  let(:correct_password) { 'password' }
+  let(:incorrect_password) { 'incorrectpassword' }
+
+  before do
+    sign_up_for_account(email: 'george@gov.uk')
```

Comment:
Better to use `sign_in_user(user)` here as it bypasses the fields and also means we don't need to user the notification spies.

Eg:
```
let(:user) { create(:user, :confirmed, password: "whatever", password_confirmation: "whatever") }

before do
  sign_in_user(user)
end
```